client/components/Content/index.js

```diff
@@ -0,0 +1,2 @@
+import Content from "./Content";
+export default Content;
```

Comment:
Do components need their own directories?

I think it's simpler to just use 

```
components/
  - Content.js
  - Content.css
  - Header.js
  - Header.css
```

Instead of creating a separate directory per component, which seems unnecessary, just keep all the files flat. And these dummy `index.js` files won't be needed. Also when the component exports something other than `default`, it has to be added in `index.js` as well.